### ISSUE TITLE
Presentation Background Item does not refresh automatically

### ISSUE THREAD

**Issue by robb-price** [Rise-Vision/rise-vision-apps#377]
<b>How it works now:</b>
1. Upload an image to Storage, and add it image as a Presentation background Image in the Presentation Background settings. 
2. Delete the image, and notice that the Presentation does not update to reflect the deletion on the DIsplay, or in the Editor.
3. Another test is set the image as the Presentation background image, and then replace the image in Storage. Notice that it does not update on the Display, but will update the Image in an Image Widget automatically.

<b>How it should work:</b> 
The Presentation background image setting should have the same refresh settings as the Image Widget.

**Comment by robb-price** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Issue 377 - Presentation Background Item does not refresh automatically](https://trello.com/c/rRh7RxdY/3656-issue-377-presentation-background-item-does-not-refresh-automatically)

**Comment by robb-price** [comment]
Our estimation is that adding the refresh mechanism would increase our server costs in ~$400/month. Also, this has been the behaviour since we implemented Viewer and it is the first complaint we have, so we can consider it is not a common problem.

Given the above, we concluded that this is not worth pursuing right now. We can work on it again once we improve our refresh mechanism to a more efficient mechanism, which is on the roadmap for Delivery.

As workarounds, users can either force a refresh of the Presentation by making a change to the Presentation or restarting the display, or they can replace the Presentation background to an Image Widget of the size of the Presentation.
